docs/Manage-Documents.rst

```diff
@@ -2082,6 +2091,47 @@ Multiple file sharing conditions
    file is already attached.
 
    |image167|
+   
+.. _AS_Files_Classification:   
+
+Activity stream file's classification
+````````````````````````````````````````
+
+When sharing multiple files in the activity stream by uploading them from your hard disk
+either by dragging and dropping or uploading them manually, you can classify them and choose
+a destination folder in eXo Documents to store them.
+
+For that purpose, you just need to click on ``Select destination folder`` link.
+
+|dest_folder|
+
+A popup of the documents drives appears allowing you to select a destination folder.
+
+.. note:: When uploading files from a space, the popup list folders of the space's drive, otherwise, it's your **Personal Documents**.
```

Comment:
The default destination folder changes switch displayed activity stream:
* Space activity stream: the space home folder is displayed
*  Home activity stream: the user personal folder is displayed.